packages/redux/src/users/actionTypes.js

```diff
@@ -1,28 +1,25 @@
 /**
- * @module profile/actionTypes
- * @category Profile
+ * @module users/actionTypes
+ * @category Users
  * @subcategory Actions
  */
 
-/** Action type dispatched when the get profile request fails. */
-export const GET_PROFILE_FAILURE =
-  '@farfetch/blackout-client/GET_PROFILE_FAILURE';
-/** Action type dispatched when the get profile request starts. */
-export const GET_PROFILE_REQUEST =
-  '@farfetch/blackout-client/GET_PROFILE_REQUEST';
-/** Action type dispatched when the get profile request succeeds. */
-export const GET_PROFILE_SUCCESS =
-  '@farfetch/blackout-client/GET_PROFILE_SUCCESS';
+/** Action type dispatched when the get users request fails. */
+export const GET_USER_FAILURE = '@farfetch/blackout-client/GET_USER_FAILURE';
```

Comment:
Just to understant if this is the right scope - your PR states that redux actions are changing to the new naming strategy, but the actions are still named "GET" instead of "FETCH"; is it supposed?

Reply:
No, I've changed them as well, thanks for the feedback

Reply:
Well spotted, changed those as well.